BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -1822,6 +1824,14 @@ unsafe private FloatCameraImage CalcZImage()
             return depthData;
         }
 
+        private FloatCameraImage CalcDistanceImage()
+        {
+            FloatCameraImage zImage = CalcZImage();
+            ProjectiveTransformationZhang projTrans = GetIntrinsics(ChannelNames.ZImage) as ProjectiveTransformationZhang;
```

Comment:
If the distance channel is used, it will typically be calculated for every frame.
This looks like a slow operation (involving a call to the RealSense API) which should be cached (preferably in GetIntrinsics).

Reply:
... I guess that request should go into a new issue. I'll create it later.

Reply:
seems reasonable